pkg/accounts/model/account.ts

```diff
@@ -14,6 +16,48 @@ export type AccountStatus = 'active' | 'notActivated';
 export type AccountFrozen = 'frozen' | 'normal';
 export type AccountSilenced = 'silenced' | 'normal';
 
+export const AccountNameSchema = z
+  .string()
+  .refine((s) => {
+    const parts = s.split('@', 3);
```

Comment:
```suggestion
    const parts = s.split('@');
```
分割数に制限をかけると, `"@hoge@example.com@false"` のようなパターンでも検査に通過してしまうと思います. こちらのような文字列もテストケースに追加しておいてください.